### PR TITLE
feat: claim nft

### PR DESCRIPTION
**Pull request by tomquirk**
Closes https://github.com/jbx-protocol/juice-interface/issues/4233

Screenshot 2024-01-26 at 7 57 38 AM

### CODE REVIEW

File: src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectCard/components/TokensPerEth.tsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+import { V2V3_CURRENCY_ETH } from 'utils/v2v3/currency'
+import { useTokensPerEth } from '../../../hooks/useTokensPerEth'
 
 export const TokensPerEth = ({
   currencyAmount,
 }: {
   currencyAmount:
     | {
-        amount: number
+        amount: string
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectCard/components/TokensPerEth.tsx:11]: pretty sure this was also getting passed a string, but maybe an `any` type made the compiler happy

File: src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectCard/components/TokensPerEth.tsx [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
         currency: V2V3CurrencyOption
       }
     | undefined
 }) => {
   const { currencyText, receivedTickets, receivedTokenSymbolText } =
-    useTokensPerEth(currencyAmount)
+    useTokensPerEth({
+      amount: parseFloat(currencyAmount?.amount || '1'),
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/ProjectDashboard/components/PayProjectCard/components/TokensPerEth.tsx:18]: account for 'no input value' here instead.

Will return as-if 1 ETH

Screenshot 2024-01-26 at 8 12 53 AM

File: src/components/v2v3/V2V3Project/ProjectDashboard/hooks/useTokensPerEth.ts [hunk 3]
```diff
@@ -53,16 +50,7 @@ export function useTokensPerEth(
...
-    if (weiPayAmt.gt(0)) {
-      return formatReceivedTickets(weiPayAmt)
-    }
-    return formatReceivedTickets(
-      (currency === V2V3_CURRENCY_ETH
-        ? utils.parseEther('1')
-        : converter.usdToWei('1')) ?? BigNumber.from(0),
-    )
-  }, [converter, currency, formatReceivedTickets, weiPayAmt])
+  const receivedTickets = formatReceivedTickets(weiPayAmt)
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/ProjectDashboard/hooks/useTokensPerEth.ts:53]: nuke all this and shift it to the component above

Comment (tomquirk) [reply src/components/v2v3/V2V3Project/ProjectDashboard/hooks/useTokensPerEth.ts:53]: https://github.com/jbx-protocol/juice-interface/pull/4235/files#r1467018318

**Comment by johnnyd-eth** [comment]
All looks great sir @aeolianeth, just got a few compiler issues:

Screen Shot 2024-01-26 at 9 16 05 am

**Pull request merged**